src/layouts/SiteLaunchPad/SiteLaunchPad.tsx

```diff
@@ -0,0 +1,159 @@
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalBody,
+  Text,
+} from "@chakra-ui/react"
+import {
+  Button,
+  Checkbox,
+  Link,
+  ModalCloseButton,
+} from "@opengovsg/design-system-react"
+import { useState } from "react"
+
+import { useSiteLaunchContext } from "contexts/SiteLaunchContext"
+
+import { SiteViewLayout } from "layouts/layouts"
+
+import { SiteLaunchStatusProps, SITE_LAUNCH_PAGES } from "types/siteLaunch"
+
+import {
+  SiteLaunchDisclaimerBody,
+  SiteLaunchDisclaimerTitle,
+} from "./components/SiteLaunchDisclaimer"
+import {
+  SiteLaunchInfoGatheringTitle,
+  SiteLaunchInfoGatheringBody,
+} from "./components/SiteLaunchInfoGathering"
+
+const RiskAcceptanceModal = ({
+  isOpen,
+  setPageNumber,
+}: {
+  isOpen: boolean
+  setPageNumber: (number: number) => void
+}): JSX.Element => {
+  const [isDisabled, setIsDisabled] = useState(true)
+  return (
+    <Modal
+      isOpen={isOpen}
+      onClose={() => setPageNumber(SITE_LAUNCH_PAGES.INFO_GATHERING)}
+    >
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader>
+          <Text textStyle="h4">
+            Check if you have full control over your DNS
```

Comment:
Sure + I think a11y is impt to care about. 
However, I dont get how <Text textStyle="h4"> not align with a11y. Is this because when using the textStyle prop to apply heading styles, the resulting HTML markup does not include a proper heading tag (e.g. <h1>, <h2>, etc.)?

Reply:
as disucssed on slack, using <Text as="h4" ...> instead